alcs-frontend/src/app/features/meeting/meeting-overview/meeting-overview.component.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject, takeUntil } from 'rxjs';
+import { ApplicationDocumentService } from '../../../services/application/application-document/application-document.service';
+import { BoardService, BoardWithFavourite } from '../../../services/board/board.service';
+import { UpcomingMeeting, UpcomingMeetingBoardMapDto } from '../../../services/meeting/meeting.dto';
+import { MeetingService } from '../../../services/meeting/meeting.service';
+
+type MeetingWithApplications = { meetingDate: number; applications: UpcomingMeeting[] };
+
+type BoardWithDecisionMeetings = {
+  boardTitle: string;
+  boardCode: string;
+  isFavourite: boolean;
+  pastMeetings: MeetingWithApplications[];
+  upcomingMeetings: MeetingWithApplications[];
+  nextMeeting: MeetingWithApplications | undefined;
+};
+
+@Component({
+  selector: 'app-meeting-overview',
+  templateUrl: './meeting-overview.component.html',
+  styleUrls: ['./meeting-overview.component.scss'],
+})
+export class MeetingOverviewComponent implements OnInit, OnDestroy {
+  destroy = new Subject<void>();
+  private boards: BoardWithFavourite[] = [];
+  private meetings: UpcomingMeetingBoardMapDto | undefined;
+  dateFormat = 'ddd YYYY-MM-DD';
```

Comment:
Is it worse moving this to the moment file? so all date formats will be in one place

Reply:
For now this is only used in this place I think its okay to leave it here, I did rename it to customDateFormat to make it obvious its not the same.